taxi/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
```

Comment:
Place changes inside of the `styles.css` to show that static files loaded successfully in your project.